components/brave_ads/browser/ad_notification_util.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
2021 everywhere

Reply:
Fixed for new files. Modified files have been left.